src/components/CourseDetails.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { COURSE_MODEL } from "../constants/CourseModel";
+import { TagList } from "./TagList";
+import { Tag } from "./Tag";
+
+const CourseDetails = ({ course }) => {
+    return (
+        <TagList>
+            <Tag icon="calendar" mainText={`Starts ${course.yearOfEntry}`} />
```

Comment:
Do we want to not show these tags when they're not present in the results?
I think they probably will be in all cases for this, but might not be for every piece of metadata. We could apply this by doing something like
```suggestion
            { course.yearOfEntry && <Tag icon="calendar" mainText={`Starts ${course.yearOfEntry}`} />}
```
This is better for this component in the long run, however it will break TagList if we have a course with no metadata - TagList requires a list of nodes as children.

Reply:
I quite like Richard's suggestion. Another approach might be to specialise the `<Tag>` with something like
```
const StartDateTag = ({ yearOfEntry }) => {
    if (yearOfEntry) {
        return <Tag icon="calendar" mainText={`Starts ${course.yearOfEntry}`} />
    }
}
```
(that's probably not totally legal syntax but just a general idea) - and then call it with
```
<TagList>
    <StartDateTag yearOfEntry={course.yearOfEntry} />
</TagList>
```

Reply:
Also there's no problem modifying `<TagList>` to NOT require children, if needed, for the time being

Reply:
Also sounds good, though I think it's worth saying that `startDateTag` and other tags could sit inside `CourseDetails` while they're so simple.

Reply:
Agreed - they can be `private` (i.e. non-exported) component(s) in `CourseDetails.js`